packages/utilities/src/focus.ts

```diff
@@ -386,7 +386,7 @@ export function isElementTabbable(element: HTMLElement, checkTabIndex?: boolean)
       element.tagName === 'TEXTAREA' ||
       isFocusableAttribute === 'true' ||
       isTabIndexSet ||
-      (element.getAttribute && element.getAttribute('role') === 'button'));
+      (element.getAttribute && element.getAttribute('role') === 'button' && isTabIndexSet));
```

Comment:
`isTabIndexSet` equates to tab index being a value of 0 or greater.

If it is true, line 389 (the new line) will be short circuited.

if you added a test to stress this scenario first you'd probably find this out. :)

Reply:
I removed the check for role=button in this function.  Setting a role=button on a div by itself doesn't make it tabbable without setting the tabIndex.  The test for checking role=button should return false as well, unless I'm doing something wrong.  I set up a basic codepen with a div with role=button and it didn't allow it to be tabbable (tested in Edge and Chrome) without a tabIndex.